packages/api/src/controllers/webhook.ts

```diff
@@ -287,9 +295,45 @@ const requestsFieldsMap: FieldsMap = {
   statusCode: `webhook_response.data->'response'->>'status'`,
 };
 
+app.put("/:id/log/:requestId/resend", authorizer({}), async (req, res) => {
+  const webhook = await db.webhook.get(req.params.id);
+  const webhookResponse = await db.webhookResponse.get(req.params.requestId);
+  await checkRequest(req, webhook, webhookResponse);
+
+  await resendWebhook(webhook, webhookResponse);
+  res.status(202).end();
+});
+
+app.get("/:id/log/:requestId", authorizer({}), async (req, res) => {
+  const webhook = await db.webhook.get(req.params.id);
+  const webhookResponse = await db.webhookResponse.get(req.params.requestId);
```

Comment:
Something funny here is that the `id` and `requestId` params are redundant right? I wonder if we'll ever need an API which is just `/webhook/log/:requestId` instead, but let's keep it like this for now I think it's consistent with standards

Reply:
Yeah I wasn't too sure which way was preferable